services/acceptance/services/registration.test.js

```diff
@@ -12,13 +12,17 @@ describe('Registration service acceptance tests', () => {
         url: h.getUrl('/registration'),
         method: 'POST',
         json: true,
+        resolveWithFullResponse: true,
         body: {
           pubKey: 'pretend-pubkey',
           curve: 'secp256k1'
         }
       })
-        .then(res => assert.ok(res.uuid, 'Missing a uuid'))
-        .catch(res => assert.equal(res.statusCode, 201));
+        .then(res => {
+          expect(res.statusCode).toEqual(201);
+          expect(res.body.uuid).toBeTruthy();
+        })
+        .catch((err) => assert.fail(err));
```

Comment:
I guess we cannot indeed use something like:

```js
expect(request({
        url: h.getUrl('/registration'),
        method: 'POST',
        json: true,
        body: {
          pubKey: 'pretend-pubkey'
        }
      }).toThrow(BLAH); 
```

Because there seems to be no way to express the check instead of `BLAH` that `err.statusCode == 400`